lib/yt/collections/authentications.rb

```diff
@@ -40,8 +40,9 @@ def next_page
       end
 
       def expected?(error)
-        error.kind == 'invalid_grant'
+        error.kind == 'invalid_grant' &&
+        !(error.description == "Code was already redeemed." || error.description == "Invalid code.")
```

Comment:
Can you make a variable or method that returns `["Code was already redeemed.", "Invalid code."]`?

Then the code will be shorter (you can use `in?` or `include?`)